rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -191,7 +227,7 @@ struct network_thread
 		{
 			// Wait with 1ms timeout
 #ifdef _WIN32
-			::WSAPoll(fds, socklist.size(), 1);
+			WSAPollEx(fds, socklist.size(), 1);
```

Comment:
Wrap it around reader_lock lock(id_manager::g_mutex) maybe, only this one, and revert my idm::get suggestion to check_unlocked.